scripts/publish.js

```diff
@@ -0,0 +1,59 @@
+#!/usr/bin/env node
```

Comment:
The publish script is limited to creating a git tag and pushing the artifact to npm. Because of how the project version is used across the code, we don't handle updating the project version here (for now).